packages/react-native/android/src/main/java/backtrace/library/BacktraceReactNative.java

```diff
@@ -49,29 +49,42 @@ public String getName() {
     @ReactMethod(isBlockingSynchronousMethod = true)
     public Boolean initialize(String minidumpSubmissionUrl, String databasePath, ReadableMap readableAttributes, ReadableArray attachmentPaths) {
         Log.d(this.NAME, "Initializing native crash reporter");
-
-        String handlerPath = context.getApplicationInfo().nativeLibraryDir + _crashpadHandlerName;
-
-        if (!(new File(handlerPath).exists())) {
-            Log.d(this.NAME, "Crashpad handler doesn't exist");
+        CrashHandlerConfiguration crashHandlerConfiguration = new backtraceio.library.models.nativeHandler.CrashHandlerConfiguration();
+        if (!crashHandlerConfiguration.isSupportedAbi()) {
+            Log.d(this.NAME, "Unsupported ABI detected.");
             return false;
         }
+        String handlerPath = context.getApplicationInfo().nativeLibraryDir + _crashpadHandlerName;
+ 
         HashMap<String, Object> attributes = readableAttributes.toHashMap();
 
         String[] keys = attributes.keySet().toArray(new String[0]);
         String[] values = attributes.values().toArray(new String[0]);
-
-        Boolean result = BacktraceDatabase.initialize(
-                minidumpSubmissionUrl,
-                databasePath,
-                handlerPath,
-                keys,
-                values,
-                attachmentPaths.toArrayList().toArray(new String[0]),
-                false,
-                null
-        );
-
+        BacktraceCrashHandlerWrapper nativeCommunication = new BacktraceCrashHandlerWrapper();
+
+        Boolean result =
+                new File(handlerPath).exists()
```

Comment:
can you add a comment that describes the difference between `initializeCrashHandler` and `initializeJavaCrashHandler`?

Reply:
done